### PR TITLE
Send reminders for experts to help companies in need

### PR DESCRIPTION
**Pull request by ArnoHolo**

### CODE REVIEW

File: app/models/selected_assistance_expert.rb [hunk 1]
```diff
@@ -10,7 +10,14 @@ class SelectedAssistanceExpert < ApplicationRecord
 
   scope :not_viewed, (-> { where(expert_viewed_page_at: nil) })
   scope :of_expert, (->(expert) { joins(:assistance_expert).where(assistances_experts: { expert: expert }) })
-  scope :of_diagnoses, (proc do |diagnoses|
+  scope :of_diagnoses, (lambda do |diagnoses|
     joins(diagnosed_need: :diagnosis).where(diagnosed_needs: { diagnosis: diagnoses })
   end)
+  scope :with_status, (->(status) { where(status: status) })
+  scope :updated_before_five_days_ago, (-> { where('updated_at < ?', 5.days.ago) })
```
Comment (Apemb) [inline app/models/selected_assistance_expert.rb]: Why five days ? why not a full week ? 
And wouldn't be clearer to write `updated_more_than_five_days_ago` ?

File: spec/models/selected_assistance_expert_spec.rb [hunk 2]
```diff
@@ -54,5 +64,85 @@
...
+      end
+    end
+
+    describe 'updated_before_five_days_ago' do
+      subject { SelectedAssistanceExpert.updated_before_five_days_ago }
+
+      let!(:selected_ae_updated_two_weeks_ago) { create :selected_assistance_expert, updated_at: 2.weeks.ago }
+
+      before { create :selected_assistance_expert, updated_at: 4.days.ago }
+
```
Comment (Apemb) [inline spec/models/selected_assistance_expert_spec.rb]: why the before and the let! ? this seems to be a bit inconsistent

File: app/services/expert_reminder_service.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class ExpertReminderService
+  attr_accessor :experts_hash
+
+  class << self
+    def send_reminders
+      @experts_hash = {}
+      selected_assistances_experts_needing_taking_care_update
```
Comment (Apemb) [inline app/services/expert_reminder_service.rb]: I would rather have methods that return the hash into send°reminders 
I don't like collateral modifications like those

File: app/services/expert_reminder_service.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      end
+    end
+
+    def init_expert_hash(selected_assistance_expert)
+      expert = selected_assistance_expert.assistance_expert.expert
+      @experts_hash[expert.id] = {
+        expert: expert,
+        selected_assistances_experts_hash: {
+          needing_taking_care_update: [],
+          with_noone_in_charge: []
```
Comment (Apemb) [inline app/services/expert_reminder_service.rb]: no_one (no one us version) vs noone ?

Comment (Apemb) [reply app/services/expert_reminder_service.rb]: or no-one (uk version)

File: app/models/selected_assistance_expert.rb (hunk 1 above)
Comment (ArnoHolo) [reply app/models/selected_assistance_expert.rb]: Five days = not too short, not spamming the expert ; but not too long, not waiting two weeks before contacting it. Well that's a try.
Ok for `updated_more_than_five_days_ago`

File: spec/models/selected_assistance_expert_spec.rb (hunk 2 above)
Comment (ArnoHolo) [reply spec/models/selected_assistance_expert_spec.rb]: You need to use `let!(:foo) { bar }` when you need to reuse `foo` elsewhere

File: app/services/expert_reminder_service.rb (hunk 3 above)
Comment (ArnoHolo) [reply app/services/expert_reminder_service.rb]: I'm OK if you're motivated to improve it :D

File: app/services/expert_reminder_service.rb (hunk 4 above)
Comment (ArnoHolo) [reply app/services/expert_reminder_service.rb]: I don't know which one is the best, what do you prefer?

Comment (Apemb) [reply app/services/expert_reminder_service.rb]: simplified english 😀 (us version !)

**Pull request merged**